mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -286,6 +273,7 @@ public boolean apply(MailboxId mailboxId) {
                     Mailbox currentMailbox = mailboxMapper.findMailboxById(mailboxId);
                     return belongsToCurrentUser(currentMailbox, mailboxSession);
                 } catch (MailboxException e) {
+                    mailboxSession.getLog().error("Can not retrieve mailboxPath associated with {0}", mailboxId, e);
```

Comment:
Ok, but I really don't like this format, blending parameters used for string formating & exception is not really great, IMHO.
slf4j has changed its behavior on that, nothing say that they will go back; and we may discover this only in production: we have to be safe on those methods & we will never speak about that.